src/app/shared/directives/course-plate.directive.ts

```diff
@@ -0,0 +1,28 @@
+import { Directive, ElementRef, HostListener, Input, OnInit, Renderer2 } from '@angular/core';
+
+@Directive({
+  selector: '[coursePlate]'
+})
+export class CoursePlateDirective implements OnInit{
+  @Input('coursePlate') public date: Date;
+
+  constructor(private element: ElementRef, private render: Renderer2) {}
+
+  ngOnInit(): void {
+    this.changeBorder();
+  }
+
+  public changeBorder(): void {
+    const currentDate = +Date.now();
+    const creationDate = +new Date(this.date);
+    const milisecondsPerDay = 86400000;
+    const isFresh = creationDate < currentDate && creationDate >= (currentDate - 14 * milisecondsPerDay)
+    if (isFresh) {
+      this.render.addClass(this.element.nativeElement, 'fresh');
+    } 
+    if (creationDate > currentDate) {
```

Comment:
use else, you don't need to check two conditions. If one is true the cond is obviously false

Reply:
done